assets/schema_input.json

```diff
@@ -162,23 +162,9 @@
                 "format": "file-path",
                 "exists": true
             },
-            "vcf": {
-                "errorMessage": "VCF file for reads 1 cannot contain spaces and must have extension '.vcf' or '.vcf.gz'",
-                "anyOf": [
-                    {
-                        "type": "string",
-                        "pattern": "^\\S+\\.vcf(\\.gz)?$"
-                    },
-                    {
-                        "type": "string",
-                        "maxLength": 0
-                    }
-                ],
-                "format": "file-path",
-                "exists": true
-            },
             "variantcaller": {
-                "type": "string"
+                "type": "string",
+                "meta": ["variantcaller"]
```

Comment:
why was this not meta?
Do we need this if we're not annotating?

Reply:
@FriederikeHanssen ?

Reply:
We need to keep track of this to organise output: `annotation/strelka/....` and organise internally otherwise we can't keep up with having multiple vcf files per sample

Reply:
ok, so not really needed.
I'm thinking we might want to keep it as it's not hurting, as it could be useful for future features